File: demo/CustomCrashHandler/ViewController.m
//
//  ViewController.m
//  CustomCrashHandler
//
//  Created by hewig on 6/30/14.
//  Copyright (c) 2014 fourplex. All rights reserved.
//

#import "ViewController.h"
#import "FPCrashHandler.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    self.title = @"Demo";
    self.view.backgroundColor = [UIColor whiteColor];

    UIAlertController *alert = [FPCrashHandler debugOptionsAlert];
    [self presentViewController:alert animated:YES completion:nil];
}

- (void)viewWillAppear:(BOOL)animated
{
    [super viewWillAppear:animated];
}

@end

File: demo-swift/CustomCrashHandler/ViewController.swift
//
//  ViewController.swift
//  CustomCrashHandler
//
//  Created by hewig on 3/12/17.
//  Copyright © 2017 Fourplex Labs. All rights reserved.
//

import UIKit
import FPCrashHandler

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        self.title = "Demo"
        self.navigationController?.present(FPCrashHandler.debugOptionsAlert(), animated: true, completion: nil)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}

